.github/workflows/lint-docs.yaml

```diff
@@ -0,0 +1,21 @@
+name: Check Markdown links
+
+on:
+  push:
+    branches:
+      - add-deadlink-workflow/furkat
```

Comment:
Note: this is for testing only, will be removed once it is agreed this change can be merged
/hold

Reply:
Removed it